eng/pipelines/aggregate-reports.yml

```diff
@@ -100,6 +118,7 @@ jobs:
       parameters:
         Directory: ""
         CheckLinkGuidance: $true
+      condition: succeededOrFailed()
```

Comment:
I don't think this condition actually works on a template inclusion. It would have to go into the template step itself.

Reply:
I will perform a test to see if it is working as expected.

Reply:
Will move the changes to azure-sdk-tool verify-link.yml.